ts/components/helpers/withErrorModal.tsx

```diff
@@ -0,0 +1,160 @@
+import { Button, H2, Text } from "native-base";
+import * as React from "react";
+import { Image, Modal, StyleSheet, View } from "react-native";
+import { connect } from "react-redux";
+
+import I18n from "../../i18n";
+import { Action, ReduxProps } from "../../store/actions/types";
+import { GlobalState } from "../../store/reducers/types";
+import variables from "../../theme/variables";
+
+export type WithErrorModalInjectedProps = {
+  error: string | undefined;
```

Comment:
Should we be using an `Option` here, since errors [are already using them](https://github.com/teamdigitale/italia-app/blob/aef79405c745aee00073dcc64de1baaea0c5dd17/ts/store/reducers/error.ts#L15)?

Reply:
Yes, Option is better. Thanks.